mmv1/templates/terraform/examples/alloydb_secondary_instance_basic.tf.erb

```diff
@@ -0,0 +1,69 @@
+resource "google_alloydb_cluster" "primary" {
+  cluster_id = "<%= ctx[:vars]['alloydb_primary_cluster_name'] %>"
+  location   = "us-central1"
+  network    = google_compute_network.default.id
+  cluster_type = "PRIMARY"
+}
+
+resource "google_alloydb_instance" "primary" {
+  cluster       = google_alloydb_cluster.primary.name
+  instance_id   = "<%= ctx[:vars]['alloydb_primary_instance_name'] %>"
+  instance_type = "PRIMARY"
+
+  machine_config {
+    cpu_count = 2
+  }
+
+  depends_on = [google_service_networking_connection.vpc_connection]
+}
+
+resource "google_alloydb_cluster" "secondary" {
+  cluster_id   = "<%= ctx[:vars]['alloydb_secondary_cluster_name'] %>"
+  location     = "us-east1"
+  network      = google_compute_network.default.id
+  cluster_type = "SECONDARY"
+
+  continuous_backup_config {
+    enabled = false
+  }
+
+  secondary_config {
+    primary_cluster_name = google_alloydb_cluster.primary.name
+  }
+
+  deletion_policy = "FORCE"
+
+  depends_on = [google_alloydb_instance.primary]
+}
+
+resource "google_alloydb_instance" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
Can you include some more coverage for different values in these tests? For example, will a secondary instance derive its `availability_type` or `query_insights_config` from its parent?

Reply:
Can add more coverage, but from what I know each instance has its own field that is not derived from the parent.
Keeping it unresolved for now. I'll confirm once again after looking at the implementation.

Reply:
Added test coverage for the fields